app/presenters/nsm/check_answers/application_status_card.rb

```diff
@@ -25,14 +26,23 @@ def row_data
           (if response.any?
              {
                head_key: 'laa_response',
-               text: join_strings(*response, *edit_links, appeal_info, *update_claim)
+               text: join_strings(*response, *edit_links, appeal_info,
+                                  *(claim.sent_back? && !claim.pending_further_information ? update_claim : []))
```

Comment:
this feels overloaded - would it be possible to move that up into `update_claim` and return [] if `claim.sent_back? && !claim.pending_further_information`? (maybe not possible if `update_claim` is used somewhere else...)

Reply:
as discussed in the dev catchup, there is a preference for showing this logic here so we can understand at a high level what's being joined